docker/zap-baseline.py

```diff
@@ -271,23 +330,119 @@ def main(argv):
                     in_progress_issues[issue["id"]] = issue
 
     if running_in_docker():
```

Comment:
IMO we should also allow using the automation framework when we are not `running_in_docker()` (and start a container ourselves, as was done before).

Reply:
We do allow the AF to be used outside of docker, but then you wont be using the packaged scans :)